internal/host/pool_host_statemachine.go

```diff
@@ -98,7 +98,7 @@ func NewPoolHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler)
 		PostTransition:   th.PostRefreshHost(statusInfoDiscovering),
 	})
 
-	var hasMinRequiredHardware = stateswitch.And(If(HasMinValidDisks), If(HasMinCPUCores), If(HasMinMemory), If(IsPlatformValid))
+	var hasMinRequiredHardware = stateswitch.And(If(HasMinValidDisks), If(HasMinCPUCores), If(HasMinMemory), If(IsPlatformNetworkSettingsValid))
```

Comment:
doesn't look like it belong into min required hardware, but i don't see that we have any network specific section, maybe we should rename this or add it separately?

Reply:
In the host/statemachine.go I moved it to the `isSufficientForInstall`
I think that the validation isn't relevant for a host in a pool, once it will get attached to a cluster the network settings become relevant

Reply:
We don't have network configuration there so it make sense.